cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -139,6 +137,8 @@ public static ConsentLibBuilder newBuilder(Integer accountId, String property, I
         onConsentUIFinished = b.onConsentUIFinished;
         shouldCleanConsentOnError = b.shouldCleanConsentOnError;
 
+        mCountDownTimer = getTimer(defaultMessageTimeOut);
```

Comment:
Also we should consider closing webview once we are throwing timeout error

Reply:
@Suryakantj , on any error we sent a callback for closing the current view:
private void onErrorTask(ConsentLibException e){
        this.error = e;
        if(shouldCleanConsentOnError) {
            storeClient.clearIABConsentData();
        }
        cancelCounter();
        closeCurrentMessageView();
        runOnLiveActivityUIThread(() -> GDPRConsentLib.this.onError.run(e));
    }